### PR TITLE
[performance](group_concat) avoid extra copy in group_concat

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run buildall

**Review by BiteTheDDDDt — DISMISSED** [review]
LGTM

File: be/src/vec/aggregate_functions/aggregate_function_group_concat.h [hunk 1]
```diff
@@ -75,6 +75,8 @@ struct AggregateFunctionGroupConcatData {
 
     std::string get() const { return data; }
```
Comment (HappenLee) [inline be/src/vec/aggregate_functions/aggregate_function_group_concat.h:76]: why not return reference？

**Comment by Mryange** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**